components/bear-market-airdrop/Blocks/Hero/Supply.tsx

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-restricted-imports */
+import { Flex, Text } from "@chakra-ui/react";
+import { FC } from "react";
+
+interface SupplyProps {
+  supply: number;
+  initialSupply: number;
+}
+
+const Supply: FC<SupplyProps> = ({ supply, initialSupply }) => {
```

Comment:
let's use named imports everywhere for all components instead of default imports

Reply:
yes, default exports = bad